Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:12
```

Comment:
I don't think a Dockerfile in the root of this repo is the right place.

I assume this is for quicktype generation. It should likely be in a separate folder like `gen`.

Reply:
Done.